misc/imp-ware.md

```diff
@@ -0,0 +1,21 @@
+# Useful applications in KDE / Steam OS
+
+- bottles
+- emudeck
+- filelight
+- flatseal
+- protontricks
+- protonup-qt
+- qBittorent
+- lutris
+- Steam Rom manager
+- Syncthing
+
+# Useful Decky Plugins for Deck
+
```

Comment:
Maybe list them in order

Reply:
addressed [here](https://github.com/leodude77/deckgame/pull/2/commits/0fff8f66fbbab9509184cf17d1404e026bf685fe#diff-20c544e38547f965e5a3cd0e9808ab6d031bbc543ae23d101495306d8cf31db0R7-R12)